Sources/MapboxCoreNavigation/PassiveLocationManager.swift

```diff
@@ -40,7 +37,9 @@ open class PassiveLocationManager: NSObject {
     /**
      The directions service that allows the location manager to access road network data.
      */
-    public let directions: Directions
+    public var directions: Directions {
```

Comment:
Do we need such variables after these changes? I would remove them or make them private

Reply:
I reverted the change

Reply:
Done, please review the latest revision.